src/spatial_ref/srs.rs

```diff
@@ -33,6 +175,24 @@ impl CoordTransform {
         })
     }
 
+    pub fn new_ex(
```

Comment:
```suggestion
    pub fn new_with_options(
```

?

Reply:
Done, but note that it's inconsistent with `Dataset::new_ex`. That one was named after `GDALOpenEx`, however, so there's some argument to be made for the inconsistency.